strongbox-commons/src/main/java/org/carlspring/strongbox/db/DbUtils.java

```diff
@@ -0,0 +1,66 @@
+package org.carlspring.strongbox.db;
+
+import com.orientechnologies.orient.core.db.ODatabaseRecordThreadLocal;
+import com.orientechnologies.orient.object.db.OObjectDatabaseTx;
+
+import java.util.function.Consumer;
+import java.util.function.Function;
+
+public class DbUtils {
+
+    public static <R> R withDatabase(Function<OObjectDatabaseTx, R> code)
+    {
+        // for simplicity use inmemory database
+        // TODO Replace it with remote instance!
+        OObjectDatabaseTx db = new OObjectDatabaseTx("memory:strongbox");
+        try
+        {
+            if (db.exists())
+            {
+                ODatabaseRecordThreadLocal.INSTANCE.set(db.getUnderlying());
+                db.open("admin", "admin");
+            }
+            else
+            {
+                db.create();
+            }
+            return code.apply(db);
+        }
+        finally
+        {
+/*            if (!db.isClosed())
```

Comment:
I see both `finally` blocks (this one and the one a bit further down below) are commented out. Is the code there needed, or not...?